packages/kbn-apm-synthtrace/src/cli/utils/synthtrace_worker.ts

```diff
@@ -123,10 +122,16 @@ async function start() {
     return Math.round(value / 1024 ** 2).toString() + 'mb';
   }
 
+  let cpuUsage = process.cpuUsage();
+
   setInterval(async () => {
-    const stats = await pidusage(process.pid);
+    cpuUsage = process.cpuUsage(cpuUsage);
```

Comment:
Q: Does this return the delta since the first call? shouldn't we just use the current cpu usage?

Reply:
From looking at the node.js docs, the diff reading to me seems more useful, as without, it is the total cpu time of the process. So to me, it makes more sense to track the diff between each logging message than just see the total CPU time being accumulated. Computing the avg usage is kind of platform dependent, particularly if trying to track % usage. Hence why I guess the dependency was used to abstract that.

Another question here might be why are we logging cpu/memory usage in the first place. Do we even need this?

Reply:
I have no idea. I don't see any problems with your change.